.circleci/config.yml

```
@@ -0,0 +1,35 @@
+version: 2.1
+
+# this allows you to use CircleCI's dynamic configuration feature
+setup: true
+
+orbs:
+  gravitee: gravitee-io/gravitee@4.1.0
+
+# our single workflow, that triggers the setup job defined above, filters on tag and branches are needed otherwise
+# some workflow and job will not be triggered for tags (default CircleCI behavior)
+workflows:
+  setup_build:
+    when:
+      not: << pipeline.git.tag >>
+    jobs:
+      - gravitee/setup_plugin-build-config:
```

You are not building a plugin but a lib that will be used by plugins.
The difference is that you will not make this lib available on download.gravitee.io.
```suggestion
      - gravitee/setup_lib-build-config:
```